ihatemoney/templates/forms.html

```diff
@@ -89,7 +89,6 @@
     {% include "display_errors.html" %}
     {{ form.hidden_tag() }}
     {{ form.file }}
-    {{ form.csrf_token() }}
```

Comment:
?

Reply:
I had a little trouble when I had to add a new form because I don't master HTML very well. That's why I created a new tab "Upload", because despite my attempts, I couldn't put it in "Settings".
But then, when I succeeded in creating this form, I had problems reading the JSON. When I searched on the internet, I was advised to add this.
It's afterwards that I realized that it didn't bring anything, so I deleted it. Especially since it was not present anywhere else.